addons/controllers/antrea/antreaconfig_controller.go

```diff
@@ -71,6 +73,12 @@ func (r *AntreaConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request
 		return ctrl.Result{}, nil
 	}
 
+	labels := antreaConfig.GetLabels()
+	if _, ok := labels[constants.PackageNameLabel]; !ok {
+		r.Log.Info(fmt.Sprintf("AntreaConfig resource '%v' does not contains package name label", req.NamespacedName))
+		return ctrl.Result{}, errors.New("AntreaConfig does not contains package name label")
```

Comment:
I suppose that the framework will retry automatically, right?

Reply:
yes